static/src/javascripts/projects/commercial/commercial-metrics.ts

```diff
@@ -7,24 +10,25 @@ const { browserId } = window.guardian.config.ophan;
 const { adBlockers } = window.guardian;
 
 let logged = false;
-const sendMetrics = (): void => {
+const initMetrics = (): void => {
 	if (logged) return;
 	const args: [string, string | undefined, boolean, boolean?] =
 		adBlockers.active === undefined
 			? [pageViewId, browserId, isDev]
 			: [pageViewId, browserId, isDev, adBlockers.active];
 
-	logged = sendCommercialMetrics(...args);
+	logged = initCommercialMetrics(...args);
 };
 
 const init = (): Promise<void> => {
-	if (!window.guardian.config.switches.commercialMetrics)
+	if (!window.guardian.config.switches.commercialMetrics) {
 		return Promise.resolve();
+	}
 
-	const userIsInSamplingGroup = Math.random() <= 0.01;
+	initMetrics();
```

Comment:
I would inline the content of this function here and get rid of the `logged` logic. There's no need to prevent it from initialising twice anymore.

Reply:
Good point! Thank you